src/orders/components/OrderDetailsPage/OrderDetailsPage.tsx

```diff
@@ -62,6 +64,13 @@ export interface OrderDetailsPageProps extends UserPermissionProps {
   }>;
   disabled: boolean;
   saveButtonBarState: ConfirmButtonTransitionState;
+  onOrderLineAdd?: () => void;
```

Comment:
Why optional?

Reply:
Callback is not needed if we're using that on normal order (not draft or unconfirmed)